data/korath/korath variants.txt

```diff
@@ -0,0 +1,707 @@
+		"Triple Plasma Core"
+		"Korath Repeater Rifle" 190
+		"Outfits Expansion" 3
+		
+		"Reverser (Comet Class)"
+		"Reverser (Lunar Class)"
+		"Steering (Lunar Class)"
+		"Thruster (Comet Class)"
+		"Thruster (Planetary Class)"
+		"Hyperdrive"
+		"Jump Drive"
+	
+	turret "Korath Banisher"
+	turret "Korath Banisher"
+	turret
+	turret "Korath Heaver"
+	turret "Korath Heaver"
+	turret "Korath Warder"
+	turret "Korath Warder"
+
+
+ship "Korath Dredger" "Korath Dredger (Hyperdrive)"
+	outfits
+		"Korath Banisher" 2
+		"Korath Grab-Strike" 3
+		"Korath Warder" 2
+		
+		"Systems Core (Large)"
+		"Triple Plasma Core"
+		"Large Heat Shunt" 3
+		"Fuel Processor"
+		"Korath Repeater Rifle" 190
+		
+		"Steering (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Thruster (Asteroid Class)"
+		"Hyperdrive"
+		"Jump Drive (Broken)"
+
+
+
+ship "Korath Raider" "Korath Raider (Crippled)"
+	sprite "ship/raider"
+	outfits
+		"Korath Grab-Strike" 2
+		"Korath Banisher"
+		"Korath Warder"
+		
+		"Triple Plasma Core"
+		"Liquid Sodium Cooler"
+		"Korath Repeater Rifle" 196
+		"Outfits Expansion"
+		
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Jump Drive (Broken)"
+		"Hyperdrive"
+	
+	turret
+	turret
+	turret "Korath Grab-Strike"
+	turret "Korath Grab-Strike"
+	turret "Korath Banisher"
+	turret "Korath Warder"
+
+
+ship "Korath Raider" "Korath Raider (Ember)"
+	sprite "ship/raider"
+	outfits
+		"Korath Grab-Strike" 4
+		"Korath Banisher"
+		"Korath Warder"
+		
+		"Triple Plasma Core"
+		"Systems Core (Medium)"
+		"Liquid Sodium Cooler"
+		"Interference Plating"
+		"Fuel Processor"
+		"Korath Repeater Rifle" 196
+		"Outfits Expansion"
+		
+		"Bow Drive (Meteor Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Jump Drive"
+		"Hyperdrive"
+	
+	turret "Korath Grab-Strike"
+	turret "Korath Grab-Strike"
+	turret "Korath Grab-Strike"
+	turret "Korath Grab-Strike"
+	turret "Korath Banisher"
+	turret "Korath Warder"
+
+
+ship "Korath Raider" "Korath Raider (Hyperdrive)"
+	sprite "ship/raider"
+	outfits
+		"Korath Grab-Strike" 4
+		"Korath Banisher"
+		"Korath Warder"
+		
+		"Triple Plasma Core"
+		"Systems Core (Medium)"
+		"Liquid Sodium Cooler"
+		"Fuel Processor"
+		"Korath Repeater Rifle" 196
+		"Outfits Expansion"
+		
+		"Bow Drive (Meteor Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Jump Drive (Broken)"
+		"Hyperdrive"
+	
+	turret "Korath Grab-Strike"
+	turret "Korath Grab-Strike"
+	turret "Korath Grab-Strike"
+	turret "Korath Grab-Strike"
+	turret "Korath Banisher"
+	turret "Korath Warder"
+
+
+ship "Korath Raider" "Korath Raider (Inferno)"
+	sprite "ship/raider"
+	outfits
+		"Korath Grab-Strike" 2
+		"Korath Digger Turret" 2
+		"Korath Inferno"
+		"Korath Warder"
+		
+		"Triple Plasma Core"
+		"Generator (Furnace Class)"
+		"Systems Core (Medium)"
+		"Liquid Sodium Cooler"
+		"Fuel Processor"
+		"Korath Repeater Rifle" 196
+		
+		"Afterburner (Comet Class)"
```

Comment:
```suggestion
```

Reply:
Sure, as discussed this makes sense, but do you have any suggestions as to what to put in its place?
I can come up with something myself when I get back from hospital, of course, but until then you have the floor :-)

Reply:
Could just add a Reverser (Comet Class)? That might sound weird on top of the Bow Drive, but it's still something to add. 

Again we will end up editing this to have the intermediate flamers instead of Infernos (I think? I remember us discussing Infernos should be saved for larger ships for both size relation and more uniqueness) and having multiple of those instead of one Inferno will hopefully alleviate the space usage issue.

Reply:
A full size reverser would require more weapon space; that's why it's only got a bow drive

While the dual inferno ship was.... awkward and terrifying, this one I think is a keeper.  Not a common build in a Raiding fleet, but seen more often defending Korath space or taking on harder Remnant missions, like the Shunt-Strike build.  Whereas the Blaze-Pike build can be deployed more generally alongside Grab-Strike builds.

Reply:
Right, took the afterburner off the base Raider, and replaced the single plasma core with an Inferno-class generator - Raiders raiding humans don't get the top of the line equipment.  5 tonnes leftover, not including the 4 tonnes I used for interference plating that I'm tempted to replaces with intrinsic interference

Reply:
Sounds good. Do what you feel is right for the plating